Dockerfile.intel-oneapi-hpckit-ubuntu20

```diff
@@ -20,6 +20,7 @@ RUN apt-get update -y && \
     intel-oneapi-inspector \
     intel-oneapi-itac \
     intel-oneapi-mpi-devel \
+    intel-oneapi-mkl-devel \
```

Comment:
This is great, I forgot to mention that the hpckit as it exists in NOAA-EMC's containers repo does not container the mkl oneapi package.  This is exactly how I'm pulling that package in.